core/src/test/scala/unit/kafka/security/auth/SimpleAclAuthorizerTest.scala

```diff
@@ -46,14 +48,18 @@ class SimpleAclAuthorizerTest extends ZooKeeperTestHarness {
     val props = TestUtils.createBrokerConfig(0, zkConnect)
     props.put(SimpleAclAuthorizer.SuperUsersProp, superUsers)
 
+    Logger.getLogger(classOf[SimpleAclAuthorizer]).setLevel(Level.DEBUG)
```

Comment:
Ah yeah, I do that sometimes to view various logging statements for testing. I removed the debug statements, so I should remove this.